config/locales/en.yml

```diff
@@ -131,6 +131,8 @@ en:
       infinite: "unlimited"
     overview:
       not_available_for_leagues: 'No overview available for leagues.'
+    schedule:
+      insert_results: "Insert result"
```

Comment:
Have you used the singular (result) here on purpose? The key name (insert_results) made me think that the localization should be „Insert results“